misconfiguration/system/ksp-zabbix-dashboard-unauthorised-access-block.yaml

```diff
@@ -0,0 +1,21 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-zabbix-dashboard-unauthorised-access-block
+  namespace: default # use your own labels here
+spec:
+  message: Access deined for zabbix.php file to non-root users
+  tags : ["zabbix", "dashboard", "unauth"]
+  selector:
+    matchLabels:
+       app: zabbix  # use your own labels here
+  file:
+    matchPatterns: 
+     - pattern: /**/zabbix/zabbix.php
```

Comment:
How one will know if the required is authorized and un-authorized, what is the confidence factor in giving this path?